lib/msf/ui/console/command_dispatcher/auxiliary.rb

```diff
@@ -70,9 +97,9 @@ def cmd_run_tabs(str, words)
   #
   # Executes an auxiliary module
   #
-  def cmd_run(*args)
+  def cmd_run(*args, cmd: 'run')
```

Comment:
To use the action kwarg, this should be:
```suggestion
  def cmd_run(*args, action: nil)
```